lib/node_loader.js

```diff
@@ -26,7 +26,7 @@ require('./http/node');
 
 // Load custom credential providers
 require('./credentials/ec2_metadata_credentials');
-require('./credentials/ecs_credentials');
```

Comment:
If you make the changes described above, you will still need to require `ecs_credentials`